scripts/trim-low-abund.py

```diff
@@ -488,6 +520,28 @@ def main():
         log_info("Saving k-mer countgraph to {graph}", graph=args.savegraph)
         ct.save(args.savegraph)
 
+    if args.summary_info is not None:
+        # when the output is stdout the name is == 1
+        if args.output is not None and args.output.name != 1:
```

Comment:
This comment is redundant at best, misleading at worst. Comment says `== 1`, code says `!= 1`.